man/chpl.rst

```diff
@@ -336,6 +336,11 @@ OPTIONS
     Enable [disable] run-time checks in safeCast calls for casts that
     wouldn't preserve the logical value being cast.
 
+**\--[no-]const-arg-checks**
+
+    Enable [disable] const argument checks (only when --warn-unstable is also
+    used)
```

Comment:
Just noting that this will have to change if you take my advice and make `--no-const-arg-checks` / `--const-arg-checks` override `--checks` / `--fast` / `--warn-unstable` etc.